src/Css.elm

```diff
@@ -3236,6 +3242,58 @@ slashedZero =
   }
 
 
+{- FEATURE TAG VALUES -}
+
+
+{-| Alias for `on` value of [`font-feature-settings`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-feature-settings#Formal_syntax)
+-}
+on : Int
+on =
```

Comment:
Ah! I completely forgot about `Html.Events.on`. So I was thinking, this seems to be an advanced feature of CSS anyway so its users probably know that `off` corresponds to 0 and `on` corresponds to 1. Do you think we can just exclude `on` and `off`?